rollup.config.js

```diff
@@ -48,16 +47,10 @@ export default {
       limit: Infinity,
     }),
     json(),
-    sass({
-      insert: false,
-      output: 'dist/styles.css',
-      options: {
-        outputStyle: 'compressed'
-      },
-      processor: css => postcss([autoprefixer])
-        .process(css)
-        .then(result => result.css)
-    }),
-
-  ]
+    postcss({
```

Comment:
Nice. I've always used postcss to supplement SASS dependencies as opposed to doing the whole compilation job itself. Simplifies things 👍

Reply:
Issue I had with other plugins (`rollup-plugin-sass` & `rollup-plugin-scss`) is that they weren't watching the Sass partials for changes (which meant the Sass wouldn't compile on the `rollup -w[atch]` task).

Maybe there's a way around this by adding another plugin to handle watching the (Sass) dependency tree? Not sure what the best option is at the mo.